modules/waf/variables.tf

```diff
@@ -0,0 +1,19 @@
+
+variable "cis_instance_id" {
+  type        = string
+  description = "CRN of the existing CIS instance."
+}
+
+variable "domain_id" {
+  type        = string
+  description = "ID of the existing domain to add a DNS record to the CIS instance."
+}
+
+variable "waf" {
```

Comment:
I think using a bool value instead of a set of string is more intuitive and avoid to add a validation step

Reply:
Updated the code